util/util.go

```diff
@@ -20,28 +20,50 @@
 package util
 
 import (
+	"os"
 	"os/user"
 	"path/filepath"
-
-	"github.com/mitchellh/go-homedir"
+	"runtime"
 )
 
+const confDir = "symfony5"
+
 func GetHomeDir() string {
-	return filepath.Join(getUserHomeDir(), ".symfony5")
+	return getUserHomeDir()
 }
 
 func getUserHomeDir() string {
+
 	if InCloud() {
 		u, err := user.Current()
 		if err != nil {
-			return "/tmp"
+			return filepath.Join(os.TempDir(), confDir)
+		}
+		return filepath.Join(os.TempDir(), u.Username, confDir)
+	}
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		home = "."
+	}
+
+	// use the old path if it exists already
+	legacy := filepath.Join(home, "."+confDir)
+	if _, err := os.Stat(legacy); !os.IsNotExist(err) {
+		return legacy
+	}
+
+	// macos only: if $HOME/.config exist, prefer that over 'Library/Application Support'
+	if runtime.GOOS == "darwin" {
+		dotconf := filepath.Join(home, ".config")
```

Comment:
this check is not strict enough and might make the `$HOME/.config` directory be used without the user "consent". If an application creates a directory at any point, the CLI will start to use the previous directory without the user knowing it.